addons/web_editor/static/src/js/editor/snippets.editor.js

```diff
@@ -1652,6 +1652,62 @@ var SnippetsMenu = Widget.extend({
         ev.stopPropagation();
         this._activateSnippet(ev.data.$snippet.parent());
     },
+    /**
+     * @private
+     * @param {Event} ev
+     */
+    _onInstallBtnClick: function (ev) {
+        var self = this;
+        var $snippet = $(ev.currentTarget).closest('[data-module-id]');
+        var moduleID = $snippet.data('moduleId');
+        var name = $snippet.attr('name');
+        new Dialog(this, {
+            title: _t(`Install "${name}"`),
+            size: 'medium',
+            $content: $('<p/>', {text: _t(`Do you want to install the "${name}" App? `)}).append(
```

Comment:
- $content will be used as `modal-body`, this cannot be a `<p/>`, there should be a wrapping div... Maybe use an xml template?
- Why is that ugly space after `App?`? :p

Reply:
Done